internal/pkg/helm/convert.go

```diff
@@ -375,3 +378,73 @@ func InstallAnsibleTasks(roleDirectory string) {
 			destinationTasksYamlFile)
 	}
 }
+
+<<<<<<< HEAD
+// Determines whether a YAML variable (i.e., "image.pullPolicy") refers to a boolean within a given Helm Chart context.
+// TODO: This is currently unused, as the testing method skips client use.  In the future, call this from cmd.go.
+func IsBooleanYamlsValue(chartClient HelmChartClient, variableName string) (bool, error){
+	variableArray := strings.Split(variableName, goTemplateMemberAccessOperator)
+
+	raw, err := chartutil.ReadValues([]byte(chartClient.Chart.Values.Raw))
+	if err != nil {
+		return false, errors.New("Could not parse " + variableName)
+	}
+
+	// Determine depth and make sure that the path that was passed has at least one element.
+	variableDepth := len(variableArray)
+	if variableDepth < 1 {
+		return false, errors.New("Cannot split " + variableName + " appropriately")
+	}
+
+	return IsBooleanYamlValue(raw, variableArray)
+}
+
+// Determines whether path within the input context refers to a boolean type.  Consulting a Helm Chart's values is
+// helpful for determining whether a Helm Chart template conditional is checking for boolean equality v.s. definition.
+func IsBooleanYamlValue(input map[string]interface{}, path []string) (bool, error) {
```

Comment:
I think the same  here, we can try to pass the pointer. Make use of struct methods.  raw is a huge string and it is copying the values and passing it.